### ISSUE TITLE
Kinetic scrolling with libinput on Linux

### ISSUE THREAD

**Issue by RatanRSur** [brave/brave-browser#6323]
## Description 

Same as this chromium issue:
https://bugs.chromium.org/p/chromium/issues/detail?id=763791

**Comment by srirambv** [comment]
Once upstream adds this we should be able to get it automatically. 

@bsclifton  @rebron  do we need to keep this issue open until upstream implements it?

**Comment by bsclifton** [comment]
@srirambv yup- that sounds good (keeping it open until it gets closed upstream)

**Comment by RatanRSur** [comment]
Is there anyone on the brave team interested in contributing this to the upstream?

**Comment by bsclifton** [comment]
@RatanRSur I don't think we can make a commitment unfortunately ☹️ Not because of lack of interest, but rather limited resources and focusing on priorities

**Comment by RatanRSur** [comment]
No problem, keep up the good work

**Comment by y4my4my4m** [comment]
@bsclifton thank you for not forgetting about this :pray:

**Comment by All3xJ** [comment]
It should be a thing,  but the issue here: https://bugs.chromium.org/p/chromium/issues/detail?id=763791 seems dead
